src/main/java/org/mqttbee/api/mqtt/MqttClientBuilder.java

```diff
@@ -59,17 +67,48 @@ public MqttClientBuilder forServerHost(@NotNull final String host) {
     }
 
     @NotNull
-    public MqttClientBuilder forServerPort(final int port) {
+    public MqttClientBuilder withServerPort(final int port) {
```

Comment:
If this is changed, should we change forServerHost too?

Reply:
done